### PR TITLE
Audit endpoints

### PR DESCRIPTION
**Pull request by jamesrichard91**

### CODE REVIEW

File: src/main/java/com/synopsys/integration/alert/web/audit/AuditEntryActions.java [hunk 1]
```diff
@@ -113,8 +113,31 @@ public AuditEntryModel get(final Long id) {
         return null;
     }
 
-    public AlertPagedModel<AuditEntryModel> resendNotification(final Long notificationdId, final Long commonConfigId) throws IntegrationException {
-        final NotificationContent notificationContent = notificationManager.findById(notificationdId).orElseThrow(() -> new AlertNotificationPurgedException("No notification with this id exists."));
+    public JobAuditModel getAuditInfoForJob(final Long jobId) {
+        if (jobId != null) {
```
Comment (gkillough) [inline src/main/java/com/synopsys/integration/alert/web/audit/AuditEntryActions.java]: What defines jobId? @bamandel may have changed this concept.

Comment (bamandel) [reply src/main/java/com/synopsys/integration/alert/web/audit/AuditEntryActions.java]: JobId will be the configurationId

File: src/main/java/com/synopsys/integration/alert/web/audit/AuditEntryActions.java [hunk 2]
```diff
@@ -113,8 +113,31 @@ public AuditEntryModel get(final Long id) {
...
+                    timeLastSent = notificationContentConverter.getContentConverter().getStringValue(auditEntryEntity.getTimeLastSent());
+                }
+                String status = null;
+                if (null != auditEntryEntity.getStatus()) {
+                    status = auditEntryEntity.getStatus();
+                }
+                return new JobAuditModel(timeCreated, timeLastSent, status);
+            }
+        }
+        return null;
```
Comment (psantos1113) [inline src/main/java/com/synopsys/integration/alert/web/audit/AuditEntryActions.java]: If we're returning null here should this method return an Optional and return an Optional.empty().  The controller would need to change to return an Optional<JobAuditModel> but what I have initially researched is that the controller returning an Optional.empty will return a 404 that the resource cannot be found which is valid here given that the jobId is a path variable indicating a specific resource.

Comment (psantos1113) [reply src/main/java/com/synopsys/integration/alert/web/audit/AuditEntryActions.java]: Would need to test the Optional returned from the controller to be sure it returns a 404

**Pull request merged**